rosie/chamber_of_deputies/tests/test_election_expenses_classifier.py

```diff
@@ -9,18 +9,28 @@
 class TestElectionExpensesClassifier(TestCase):
 
     def setUp(self):
-        self.dataset = pd.read_csv('rosie/chamber_of_deputies/tests/fixtures/election_expenses_classifier.csv',
-                                   dtype={'name': np.str, 'legal_entity': np.str})
-        self.subject = ElectionExpensesClassifier()
+        self.election_expenser_classifier = ElectionExpensesClassifier()
 
-    def test_is_election_company(self):
-        self.assertEqual(self.subject.predict(self.dataset)[0], True)
+    def test_legal_entity_is_a_election_company(self):
+        self.dataframe = self._create_dataframe([['CARLOS ALBERTO DA SILVA', 'ELEICAO 2006 CARLOS ALBERTO DA SILVA DEPUTADO', '409-0 - CANDIDATO A CARGO POLITICO ELETIVO']])
 
-    def test_is_not_election_company(self):
-        self.assertEqual(self.subject.predict(self.dataset)[1], False)
+        prediction_result = self.election_expenser_classifier.predict(self.dataframe)
 
-    def test_fit(self):
-        self.assertEqual(self.subject.fit(self.dataset), self.subject)
+        self.assertEqual(prediction_result[0], True)
 
-    def test_tranform(self):
-        self.assertEqual(self.subject.transform(), self.subject)
+    def test_legal_entity_is_not_election_company(self):
+        self.dataframe = self._create_dataframe([['PAULO ROGERIO ROSSETO DE MELO', 'POSTO ROTA 116 DERIVADOS DE PETROLEO LTDA', '401-4 - EMPRESA INDIVIDUAL IMOBILIARIA']])
```

Comment:
What about making this test a bit more readable?

```python
    def test_legal_entity_is_not_election_company(self):
        data = [[
            'PAULO ROGERIO ROSSETO DE MELO',
            'POSTO ROTA 116 DERIVADOS DE PETROLEO LTDA',
            '401-4 - EMPRESA INDIVIDUAL IMOBILIARIA'
         ]]
        self.dataframe = self._create_dataframe(data)
        prediction_result = self.election_expenser_classifier.predict(self.dataframe)
        self.assertEqual(prediction_result[0], False)

Reply:
@cuducos, I fixed the point mentioned above. :)